index.js

```diff
@@ -26,6 +26,7 @@ export { default as PasswordValidationField } from './lib/PasswordValidationFiel
 export { default as ProxyManager } from './lib/ProxyManager';
 
 export { default as SearchAndSort } from './lib/SearchAndSort';
+export { default } from './lib/SearchAndSort/components/CheckboxFilter';
```

Comment:
Same comment as on #391; I was under the impression we need to `export { default as CheckboxFilter } ... `since otherwise we're making CheckboxFilter the default export, which we don't want.